webapp/src/lib/dialog-transaction.ts

```diff
@@ -34,12 +34,21 @@ export class DialogTransactionComponent<T> implements OnInit {
     }
 
     async ngOnInit() {
-        this.bcv.updateBlocks();
+        this.updateBlocks();
+        if (!this.bcv.currentBlock) {
+            await new Promise((resolve) => {
+                this.bcv.newStatus.subscribe((update) => {
+                    if (update) {
```

Comment:
With void, no need to check for `update` as firing is enough to know that.
This promise block can be reduced to `await new Promise(this.bcv.newStatus.subscribe);` (if I'm correct).

Reply:
That gives:
```
ERROR Error: "Uncaught (in promise): TypeError: this is undefined
```
I also tried 
```
await this.bcv.newStatus.toPromise();
```
which doesn't work, neither.

Reply:
Haa sure, `this` issue; nevermind then.